spec.html

```diff
@@ -297,31 +298,31 @@ <h1>Runtime Semantics: ClassElementEvaluation</h1>
     <emu-grammar>ClassElement : <ins>DecoratorList?</ins> MethodDefinition</emu-grammar>
     <emu-alg>
       1. <ins>If |DecoratorList| is present, let _decorators_ be the result of performing DecoratorListEvaluation of |DecoratorList|.</ins>
-      1. <del>Return</del><ins>Let _elements_ be ?</ins> ClassElementEvaluation of |MethodDefinition| with arguments ! Get(_homeObject_, `"prototype"`),_enumerable_, and `"prototype"`.
-      1. <ins>If |DecoratorList| is present, for _element_ in _elements_, set _element_.[[Decorators]] to _decorators_.</ins>
-      1. <ins>Return _elements_.</ins>
+      1. <del>Return</del><ins>Let _method_ be first element of result of ?</ins> ClassElementEvaluation of |MethodDefinition| with arguments ! Get(_homeObject_, `"prototype"`),_enumerable_, and `"prototype"`.
```

Comment:
Rather than this being the point where we start to use "the first element", probably ClassFieldElementEvaluation and ClassElementEvaluation should be changed to return a single item rather than a list. (These changes would be in the proposal-class-fields and proposal-private-methods repositories).

Reply:
Sure, I will upload pr for this in those repositories.

Reply:
Great!